contents/handbook/growth/sales/customer-onboarding.md

```diff
@@ -1,37 +1,116 @@
-- Ask if you can be added to their team in the app to do account reviews
-  - If so, set up recurring task to do this (base on the size of the account)
-- Create a private channel for them in Slack (so it's easy to get hold of each other).
-  - If they work at a small company, add them to our internal Slack. The advantage of this is that they're going to be logged into it all day, so are likely to be more responsive.
-  - If they work at a huge company, they are unlikely to get a channel approved - so use PostHog Community Slack. Check with them if this is likely to be the case.
-  - If they don't want to use Slack at all, then you should get them to email hey@posthog.com with any issues.
-- Add the Papercups app to the private Slack channel
-- Book a setup call and ask the client to invite product/engineering stakeholders
-  - Cover how to do product led growth (AARRRR) so they get their events in properly
-  - Make sure everyone has seen a demo of the main functionality
-  - Get a sense of how quick/slow the implementation will be - so you know what to expect
-  - Check how the deployment is going and if any help needed
-- Expand the account
-  - If they're a large client, they may have multiple product teams. Book Quarterly Business Reviews with the customer - this is a run through of their setup, and our roadmap. You should encourage stakeholders from other products or wider parts of their business to attend too - they may be interested in rolling PostHog out.
-  - Work with the client to make sure they:
-    - (i) track product performance (they've got events coming in, they are measuring AARRR and they've got a dashboard that breaks down each high level metric)
-    - (ii) diagnose performance
-    - (iii) release changes with our platform. Often customers may be using just one of the three areas at first.
-    - file issues for feature requests if we're missing something / talk to our product team about these
-
-There are regular checks you should do in bulk across all paying accounts:
-
-- Weekly check of usage/invoice amounts
-  - do any paying customers have unexpectedly large or small usage?
-  - any issues collecting payments?
-  - any cancellations? (gulp)
-- Every two weeks do a check of latest features - share cool stuff 1/1 with every customer in Slack.
-- Every month, get our product team to discuss potentially relevant roadmap features with them, or consider if our product marketing team should run a case study with our clients (free or paid). This helps deepen the relationships we have with them.
+We believe our unique value propositon is that we provide a fully-integrated Product Data stack, allowing users to go across Analytics, Recordings and Experimentation with ease.  To that end, we want them to see this value as quickly as possible when evaluating us against other solutions.
```

Comment:
This is hilariously pedantic, but the tone of this handbook item goes from super formal to casual. I prefer the casual tone a lot more.
```suggestion
Our moat is that we have a fully-integrated tool that allows customers to go across Analytics, Recordings and Experimentation easily.  We want new customers to see the value of this as quickly as possible when evaluating us against other solutions.
```